news/2018-09-09-project-loved-week-of-september-9th.md

```diff
@@ -0,0 +1,230 @@
+
+## <a name="osu" id="osu"></a>osu!
+
+osu! Loved candidates were chosen by [Toy](https://osu.ppy.sh/users/2757689)!
+
+---
+
+[![](/wiki/shared/news/2018-09-09-project-loved-week-of-september-9th/osu/nisemono-chuuihou.jpg)](https://osu.ppy.sh/community/forums/topics/801917)
+
+[B-ko(Cv:Touyama Nao) - Nisemono Chuuihou](https://osu.ppy.sh/beatmapsets/531751) by [Musty](https://osu.ppy.sh/users/251683) et al.  
+*written by [DigitalHypno](https://osu.ppy.sh/users/4384207)*
+
+Do you ever play through the slow intro section of a map and just wish it would go faster? And faster? And **faster**? Introducing Nisemono Chuuihou, a full mapset by Musty ranging from mid 1\* to a seemingly impossible 8\*. As the song progresses through its 3:30 length, its BPM increases from already crazy speeds to even crazier speeds, not forgetting to slow down every so often for musical effect. Accompanied by the 210+ players who have this map favorited, I'm sure I'm not alone in saying that every difficulty in this mapset does an amazing job capturing the song's intensity in a way that's exciting to play.
+
+[![](/wiki/shared/news/2018-09-09-project-loved-week-of-september-9th/osu/forward.jpg)](https://osu.ppy.sh/community/forums/topics/801918)
+
+[Gohma - Forward](https://osu.ppy.sh/beatmapsets/731899) by [Evgerable](https://osu.ppy.sh/users/5657937)  
+*written by [DigitalHypno](https://osu.ppy.sh/users/4384207)*
+
+This song, Forward, definitely has a jumpy feel to it. Evgerable captured this feeling masterfully in his mapset of 5 different difficulties. The star ratings range from low 5 stars to mid 7 stars, filling every gap in-between, so no matter what skill level you're at, you're sure to find a difficulty you enjoy. The gimmick of this mapset is slightly larger CS than the norm, with most difficulties using CS 3.6 and the bottom difficulty using CS 3.2. If that sounds like your thing, come join the 200+ players who already have this mapset favorited, and be sure to vote it for loved!
+
+[![](/wiki/shared/news/2018-09-09-project-loved-week-of-september-9th/osu/ghost-rule.jpg)](https://osu.ppy.sh/community/forums/topics/801919)
+
+[DECO\*27 - Ghost Rule](https://osu.ppy.sh/beatmapsets/425169) by [val0108](https://osu.ppy.sh/users/243917), [Kenterz](https://osu.ppy.sh/users/1587976), [z1085684963](https://osu.ppy.sh/users/1816779) and [caren\_sk](https://osu.ppy.sh/users/542498)  
+*written by [DigitalHypno](https://osu.ppy.sh/users/4384207)*
+
+A song popularized by a few other mapsets in the Ranked and Loved categories, val0108's mapset of Ghost Rule with 4 different difficulties from 4 different mappers—Kenterz, val0108, z1085684963 and caren\_sk—does a great job capturing the song in styles that aren't offered by the other sets. It certainly shows, too: the mapset has over 190 favorites, and that number is sure to increase. Whether you're a fan of unconventional flow or large, flowy, full screen jumps, this mapset has something for you.
```

Comment:
```
[...] val0108's mapset of Ghost Rule with 4 different difficulties from 4 different mappers—Kenterz, val0108, z1085684963 and caren\_sk—does a great [...]
```
It seems odd to include val0108 as a different mapper when it was earlier said to be his mapset. Though, I'm not sure if there is a better way to word this.

/edit After reading down a bit, this kinda makes sense as it isn't talking about guest difficulties.